bedrock/security/templates/security/client-bug-bounty.html

```diff
@@ -0,0 +1,101 @@
+{% extends "security/base.html" %}
+
+{% block page_title %}Mozilla Security Bug Bounty Program{% endblock %}
+{% set body_id = "bug-bounty" %}
+
+{% block article %}
+  <article class="section-content" itemscope itemtype="http://schema.org/Article">
+    <header>
+      <h1 itemprop="name" class="title-shadow-box">Client Bug Bounty Program</h1>
+    </header>
+    <div itemprop="articleBody">
+      <h2>Introduction</h2>
+
+      <p>The Mozilla Client Security Bug Bounty Program is designed to encourage security research in Mozilla software and to reward those who help us create the safest Internet software in existence.</p>
+
+      <h2>General Client Bounty Guidelines</h2>
+
+      <p>Mozilla will pay a bounty for certain client security bugs,  as detailed below.  All security bugs must follow the following general criteria to be eligible:
+      </p>
+
+      <ul>
+        <li>Security bug must be original and previously unreported.</li>
+        <li>Security bug must be a remote exploit, the cause of a privilege escalation, or an information leak.</li>
+        <li>Submitter must not be the author of the buggy code nor otherwise involved
+          in its
+          contribution to the Mozilla project (such as by providing check-in
+          reviews).
+        </li>
+        <li>Employees of the Mozilla Foundation and its subsidiaries are
+          ineligible.
+        </li>
+      </ul>
+      <p>If two or more people report the bug together the
+        reward will be divided among them. </p>
+
+      <h2>Reward Guidelines</h2>
+
+      <p>Mozilla will pay a bounty for client and security bugs as detailed below. All security bugs must follow the following general criteria to be eligible:</p>
+
+       <p> Eligible security bugs may be present in any of the current main development or released versions of Firefox, Thunderbird, Firefox for Android, or FirefoxOS as released by Mozilla Corporation (e.g. Nightly mozilla-central, Firefox Developer Edition, or Beta test versions, as well as the final release product versions)</p>
+
+        <p>The security rating given by Bounty Committee for a bug must rated a "sec-critical" or a "sec-high" in order for it to be eligible for a bounty. Some "sec-moderate" bugs may be eligible for the bounty as well. (See https://wiki.mozilla.org/Security_Severity_Ratings for details of the rating qualifications.)</p>
+
+        <p>All bounties paid will be the discretion of the Mozilla Bounty Committee, which will evaluate the severity of reported issues with the help of engineers who work on the affected code. Security researchers are invited to participate in the assignment of ratings, but final decisions on the rating are at the discretion of the Bounty Committee.</p>
```

Comment:
Should the beginning of this sentence read: "All bounties paid will be **at** the discretion of the..."?